### ISSUE TITLE
Losing custom image for Rocket Chat on restart

### ISSUE THREAD

**Issue by RoiArthurB** [getferdi/ferdi#525]
**Describe the bug**
I can set a custom image during my current session for my service Rocket Chat (which is self hosted but I don't think it really matter). The image stay during that session, but when I restart Ferdi (totally closing the app and electron and restarting it) the image disappear quickly (when the page is loaded).

**To Reproduce**

1. Start Ferdi
2. Open Edit page
3. Set custom icon
4. Completely close Ferdi
5. Start again Ferdi
6. See the image disappearing after some seconds

**Expected behavior**
Leaving the icon there

**Screenshots**
Before restart :
![image](https://user-images.githubusercontent.com/16764085/78008118-7599de80-7369-11ea-8b83-45521097d626.png)

After restart : 
![image](https://user-images.githubusercontent.com/16764085/78008003-43887c80-7369-11ea-9a94-89ea9b537691.png)

**Desktop:**

- OS: Manjaro
- Ferdi Version: ferdi-git
- Server Used: None

**Additional context**
- The icon is still in `.config/Ferdi/icons`
- It works for my Slack services

**Comment by bryanasdev000** [comment]
Hi!
I have the same problem, I'm running ferdi-bin-5.5.0-1 on Arch Linux. Both the custom and normal icons disappear after the app is restarted. This behavior is repeated only for Rocket.Chat.

**Comment by yesquines** [comment]
Hello, 

I have the same issue running Ferdi 5.0.0 (x64) on Ubuntu 18.04. 

I used the normal icon and the custom. When I used the custom I tried the follow images types: **svg, png, jpg**. But, always Rocket.Chat icons disappear.

**Comment by nil0x42** [comment]
Hi, i have exactly the same issue with Ferdi 5.5.0

**Comment by ragusa87** [comment]
This issue is related to https://github.com/getferdi/ferdi/issues/780

**Comment by prigal** [comment]
Problem still present in 5.6 beta2

Moreover if your rocket.chat has already a custom icon (svg forced in Rocket admin), Ferdi try to use it and fail loading it resulting in same issue.

**Comment by bfbones** [comment]
This is a combination of two bugs:
1. the custom image is applied, but there is a function in `webview.js` to set the servers icon, there is no check if a custom image is used so it gets overwritten BUT...
2. the icon isn't loading correctly from the server due to a doubled forward slash
That results in a blank icon.
So what that means is, if your server icon is loading correctly your custom image will be still overwritten by it.
You got two options: fixing the server icon loading or removing it entirely if you want to use a custom icon, as I mentioned here: https://github.com/getferdi/ferdi/issues/940?_pjax=%23js-repo-pjax-container#issuecomment-717130861

**Comment by RoiArthurB** [comment]
Just re-uping the issue to know if it's fixed for someone ? 
Personally, I didn't re-try to put a custom icon, but it didn't reappear (so I have an empty icon 🤷 )

**Comment by bryanasdev000** [comment]
I just did some quick tests after a heads up from @juanluisbaptiste in  #780 and indeed it's working.

For NixOS folks:

```
    overlays = [
      (self: super: {
        ferdi = super.ferdi.overrideAttrs (old: rec {
          version = "5.6.0-beta.5";
          name = "${old.pname}-${version}";
          src = super.fetchurl {
            url =
              "https://github.com/getferdi/ferdi/releases/download/v${version}/ferdi_${version}_amd64.deb";
#              "https://github.com/getferdi/ferdi/releases/download/v5.6.0-beta.5/ferdi_5.6.0-beta.5_amd64.deb"
            sha256 = "sha256-fDUzYir53OQ3O4o9eG70sGD+FJ0/4SDNsTfh97WFRnQ=";
          };
        });
      })
    ];
  };
```

I still need to test the notification badges, but it's for another issue.

**Comment by RoiArthurB** [comment]
Didn't update, but the issue is fixed on version 5.6.2 (and probably earlier, but that the one I have rn and its bug free ;) )
